### PR TITLE
python.pkgs.Pyro4: new package (at version 4.74)

### PR DESCRIPTION
**Pull request by prusnak**
###### Motivation for this change

I noticed that nixpkgs contains only Pyro Python package, which is obsolete now (see https://pypi.org/project/Pyro/) and Pyro4 should be used instead.

This PR contains a package for Pyro4 including its dependency serpent.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -4684,6 +4684,12 @@ in {
 
   Pyro = callPackage ../development/python-modules/pyro { };
```
Comment (Mic92) [inline pkgs/top-level/python-packages.nix]: How about removing the old version then?

Comment (prusnak) [reply pkgs/top-level/python-packages.nix]: It seems that openopc uses that :-/ https://github.com/NixOS/nixpkgs/blob/92a047a6c4d46a222e9c323ea85882d0a7a13af8/pkgs/tools/misc/openopc/default.nix#L4

Comment (Mic92) [reply pkgs/top-level/python-packages.nix]: This project was updated last in 2015. I will propose the removal from nixpkgs.

Comment (Mic92) [reply pkgs/top-level/python-packages.nix]: https://github.com/NixOS/nixpkgs/pull/49362

Comment (Mic92) [reply pkgs/top-level/python-packages.nix]: The package was removed.

**Comment by Mic92** [comment]
@GrahamcOfBorg build python27Packages.Pyro4 python27Packages.selectors34 python27Packages.serpe
nt python36Packages.Pyro4 python36Packages.selectors34 python36Packages.serpent

**Comment by Mic92** [comment]
```
@GrahamcOfBorg build python27Packages.Pyro4 python27Packages.selectors34 python27Packages.serpent python36Packages.Pyro4 python36Packages.selectors34 python36Packages.serpent
```

**Comment by prusnak** [comment]
Thanks!

**Pull request merged**